dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventsExportControllerTest.java

```diff
@@ -312,6 +312,29 @@ void shouldGetEventWithRelationshipsWhenRelationshipIsDeletedAndIncludeDeletedIs
     assertHasMember(jsonRelationship, "bidirectional");
   }
 
+  @Test
+  void shouldGetEventWithNoRelationshipsWhenEventIsOnToEndOfAUnidirectionalRelationship() {
+    TrackedEntity from = trackedEntity();
```

Comment:
Can you use the same setup as in dhis-2/dhis-test-web-api/src/test/java/org/hisp/dhis/webapi/controller/tracker/export/enrollment/EnrollmentsExportControllerTest.java where you get a "real" relationship from the base_data?

Reply:
Not really, this test class is not using the base_data, so I would need to refactor the entire class.
I am not doing it in this PR.
Still hope to do it soon enough though